src/Exceptionless.Core/Extensions/UriExtensions.cs

```diff
@@ -10,7 +10,7 @@ public static class UriExtensions {
         }
 
         public static string ToQueryString(this IEnumerable<KeyValuePair<string, string>> collection) {
-            return collection.ToConcatenatedString(pair => pair.Key == null ? pair.Value : $"{pair.Key}={pair.Value}", "&");
+            return collection.ToConcatenatedString(pair => pair.Key == null ? pair.Value : $"{pair.Key}={System.Web.HttpUtility.UrlEncode(pair.Value)}", "&");
```

Comment:
I have adopted it to Uri.EscapeDataString method to instead of HttpUtility.UrlEncode. And I have learned it. Thanks very much. We are waiting for your moving to asp.net core.